packages/daemon/templates/wasm/predict.js

```diff
@@ -0,0 +1,41 @@
+const tvmjs = require("./tvmjs.bundle");
```

Comment:
Shall we use ESM?

Reply:
We need to make sure this file is working on our targets: browser/Node.js WASI :)

Reply:
This file is browser/Node compatible. And it is generated based on Emcc WASI implementation.

Reply:
What I mean is the file packages/daemon/templates/wasm/predict.js